source/github-events/on-field-keydown.tsx

```diff
@@ -24,5 +24,5 @@ export function onCommentFieldKeydown(callback: DelegateFieldEvent): void {
 }
 
 export function onConversationTitleFieldKeydown(callback: DelegateFieldEvent): void {
-	onFieldKeydown('#issue_title', callback);
+	onFieldKeydown('#issue_title, #pull_request_title, #saved-reply-title-field', callback);
```

Comment:
```suggestion
	onFieldKeydown('#issue_title, #pull_request_title', callback);
```
This is used in other places, we cant just add selectors here

Reply:
Makes sense. Thanks